public/index.html

```diff
@@ -1,34 +1,71 @@
 <!DOCTYPE html>
 <html lang="en">
+	<link rel="stylesheet" href="/home/victoria/myCVInHtml/public/style.css">
```

Comment:
Let's use **style.css** instead of **/home/victoria/myCVInHtml/public/style.css**